pkgs/top-level/python-packages.nix

```
@@ -852,6 +852,8 @@ self: super: with self; {
     inherit (pkgs.darwin.apple_sdk.frameworks) AudioToolbox AudioUnit CoreServices;
   };
 
+  auditok_0_1_5 = callPackage ../development/python-modules/auditok_0_1_5 { };
```

`ffsubsync` can use an override to get 0.1.5 as it's more a tool than a module. 

```suggestion
  auditok = callPackage ../development/python-modules/auditok { };
```

---

To clarify, you're suggesting I bump this package to 0.2.0 and then inline an override for 0.1.5 into the `ffsubsync` package?

This would require me to package [genty](https://pypi.org/project/genty/) first in a separate PR, since 0.2.0 depends on it for tests. I could do this, but it feels a bit weird to maintain that when it isn't used by anything.

---

You could still set the initial version to 0.1.5. With `passthru.skipBulkUpdate = true` the package would be excluded from mass-updates.